client/src/components/FeedbackForm.js

```diff
@@ -39,10 +55,18 @@ const FeedbackForm = ({ student }) => {
                     Write feedback for the student.
 				</label>
 				<textarea
-					name="message"
+					name="feedback"
 					className="feedback-message"
-					onChange={(e) => handleChange(e)}
+					onChange={handleChange("feedback")}
```

Comment:
Is this the correct way to use a handler function? I think this should be referenced without passing any arguments as it was defined earlier.

Reply:
If we are passing the input to an object property (same key), then it is the way .